project_members.go

```diff
@@ -14,6 +14,8 @@
 // limitations under the License.
 //
 
+// N.B Documentation does not support remove functionality, alternative is to disable an existing mirror.
```

Comment:
I would not add this comment here. If needed it should be placed in the comment of the related method.